private-media-framework/draft-ietf-perc-private-media-framework.md

```diff
@@ -711,3 +698,362 @@ a co-author.
+packet.
+
+When the media distributor receives a packet, it will use the HBH
+associated with the sending adjacent endpoint or peer media distributor
+to decrypt the packet.  The result of that operation will yield a
+packet that looks like the output of "Step 2" above.  The media
+distributor would then start at "Step 3" using the HBH key
+for the adjacent endpoint or media distributor to which it will
+send the media packet.
+
+Note that the EKT Field will always be present and is of no
+value to the media distributor.  However, the EKT field needs to
+be preserved as the packet is forwarded to the next endpoint or
+media distributor.  Thus, the media distributed is expected to
+remove that field before performing HBH decryption and encryption
+steps, then re-append the EKT field precisely as it was received.
+
+## PERC Packet Format
+
+While the previous sections presented a very high-level overview of the
+construction of a PERC packet, it is useful to view the entire packet
+in a familiar way.
+
+[@fig-perc-packet-format] presents a complete picture of what a PERC
+packet looks like when transmitted over the wire.  The style used
+to present the packet is borrowed from [@RFC7714] and the packet
+contains a Full EKT Field.
+
+{#fig-perc-packet-format align="center"}
+~~~
+   0                   1                   2                   3
+   0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+A |V=2|P|X|  CC   |M|     PT      |       sequence number         |
+  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+A |                           timestamp                           |
+  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+A |           synchronization source (SSRC) identifier            |
+  +=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+
+A |            contributing source (CSRC) identifiers             |
+A |                               ....                            |
+  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+A |                    RTP extension (OPTIONAL)                   |
+  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+C :                                                               :
+C :                       Ciphertext Payload                      :
+C :                                                               :
+  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+R :                                                               :
+R :                        EKT Ciphertext                         :
+R :                                                               :
+  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+R |   Security Parameter Index    | Length                        |
+  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+R |0 0 0 0 0 0 1 0|
+  +-+-+-+-+-+-+-+-+
```

Comment:
Instead of pulling out the parts of the "FullEKTField" here, I think I would rather just show a block labeled "FullEKTField"